src/MyLinkedList.java

```diff
@@ -0,0 +1,151 @@
+public class MyLinkedList
+{
+    MyNode head;
+    public void addToFront(int value)
+    {
+        MyNode node=new MyNode(value);
+        node.setNext(head);
+        head=node;
+    }
+    public MyLinkedList getSublist(int start, int end)
+    {
+        MyLinkedList sublist=new MyLinkedList();
+        MyNode current=head;
+        for(int loop_var=1;loop_var<=end;loop_var++)
+        {
+            int temp=current.getValue();
```

Comment:
I believe you don't need this temp var. Refactor this method so that you don't need this temp var.

Reply:
resolved